lib/markdown.dart

```diff
@@ -82,5 +82,6 @@ export 'src/inline_syntaxes/link_syntax.dart';
 export 'src/inline_syntaxes/soft_line_break_syntax.dart';
 export 'src/inline_syntaxes/strikethrough_syntax.dart';
 export 'src/inline_syntaxes/text_syntax.dart';
+export 'src/line.dart';
```

Comment:
changelog: document new type, too!

Also: we changed the public API of public classes here, right? `lines` field now different. should be in changelog, too!

Reply:
Fixed!